docs/src/components/footer.tsx

```diff
@@ -1,23 +1,50 @@
 import Link from 'next/link';
-
-import Navigation from './navigation';
+import { KeystaticLogo } from './navigation/keystatic-logo';
+import { SocialLinks } from './navigation/social-links';
 
 export default function Footer() {
   return (
     <footer className="bg-keystatic-gray-light pb-10">
-      <Navigation showCta={false} />
-      <div className="mx-auto mt-4 flex max-w-7xl flex-col justify-between gap-y-4 gap-x-12 px-6 text-center md:flex-row md:text-left">
-        <p className="leading-none text-keystatic-gray-dark">
-          &copy; {new Date().getFullYear()} Thinkmill. All rights reserved.{' '}
-          <Link href="/privacy-policy" className="underline hover:text-black">
-            Privacy policy
-          </Link>
+      <div className="mx-auto max-w-7xl">
+        <nav className="p-6 flex flex-col md:flex-row items-center justify-between gap-6">
+          <KeystaticLogo />
+          <SocialLinks />
+        </nav>
+      </div>
+
+      <div className="mx-auto mt-2 flex max-w-7xl flex-col justify-between gap-y-4 gap-x-12 px-6 text-center md:flex-row">
+        <p className="leading-tight text-keystatic-gray-dark text-center md:text-left">
+          <CopyrightPrivacyPolicyContent />
         </p>
-        <p className="leading-none text-keystatic-gray-dark">
+
+        <p className="leading-tight text-keystatic-gray-dark text-center md:text-right">
           Made with <span className="text-[#ff3838]">♥</span> in Australia by a
```

Comment:
Running the screen reader over this I think we should change it to ❤ so that it doesn't read out the playing card thing.

Reply:
Great catch

Reply:
Wow didn't know it did that 😅 Thank you!!